File: apps/api/src/chunking/__fixtures__/sample.py
import json


def greet(name: str) -> str:
    return f"Hello, {name}!"


class Greeter:
    def __init__(self, verbose: bool):
        self.verbose = verbose

    def greet(self, name: str) -> str:
        if self.verbose:
            print(f"Greeting {name}")
        return greet(name)

File: apps/api/src/chunking/__fixtures__/sample.ts
import { readFile } from 'node:fs/promises';

export interface Options {
  verbose: boolean;
}

export function greet(name: string): string {
  return `Hello, ${name}!`;
}

export class Greeter {
  constructor(private readonly options: Options) {}

  greet(name: string): string {
    if (this.options.verbose) {
      console.log(`Greeting ${name}`);
    }
    return greet(name);
  }
}

export const shout = (name: string): string => {
  return greet(name).toUpperCase();
};
